src/ParkingAPI.java

```diff
@@ -2,14 +2,30 @@
 
 public class ParkingAPI {
 
-    
+	public ParkingAPI() {
 
-    public ParkingAPI() {
+	}
 
-    }
+	public int getOpenSpots(ParkingLot parkingLot) {
+		Random random = new Random(parkingLot.ordinal());
+		return random.nextInt(110);
+	}
+	
+	public int getFilledSpots(ParkingLot parkingLot) {
+		Random random = new Random(parkingLot.ordinal());
+		return random.nextInt(110);
+	}
```

Comment:
Ahh, that makes sense. I tried adding a  "total parking lot spots" value to the enum and for some reason I am getting an error: 
`public enum ParkingLot {
    NORTH_LOT,SOUTH_LOT,WHITTNEY_LOT;
    
    private int totalParkingSpots;
	
	ParkingLot(int value){
		totalParkingSpots = value;
	}
	
}`

I used the Java eNums API as reference since I am new to working with eNums: https://docs.oracle.com/javase/tutorial/java/javaOO/enum.html

The

Reply:
When you create a constructor, you need to define the actual constructor in each of the variables, so it would be like:
```java
public enum ParkingLot {
    NORTH_LOT(100), SOUTH_LOT(100), WHITTNEY_LOT(20);
    private int totalParkingSpots;
    ParkingLot(int value) {
        totalParkingSpots = value;
    }
}
```

Also, for the code block you need to use the following:

\`\`\`java
code goes here
\`\`\`